recover_handler.go

```
@@ -15,11 +15,6 @@ func Recover(handler http.Handler) http.Handler {
 			defer func() {
 				if rec := recover(); rec != nil {
 					msg, stack := extractPanic(rec)
-					log.WithFields(log.Fields{
```

So if a go service panics that error will be lost without this?